pymilvus/client/grpc_handler.py

```diff
@@ -671,7 +674,7 @@ def wait_for_creating_index(self, collection_name, field_name, timeout=None, **k
 
     @retry_on_rpc_failure(retry_times=10, wait=1)
     @error_handler
-    def load_collection(self, collection_name, timeout=None, replica_number=1, **kwargs):
+    def load_collection(self, collection_name, replica_number=1, timeout=None, **kwargs):
```

Comment:
Not sure if there any compitible issues here. Fortunately `replica_number` hasn't been released.

Reply:
@longjiquan This would only influence wangting for now, I'll make sure she is notified after this PR is merged

Reply:
Got it.